packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -359,6 +359,21 @@ interface DatabaseClusterBaseProps {
    * @default - IPV4
    */
   readonly networkType?: NetworkType;
+
+  /**
+   * The Active Directory directory ID, required for setting up Kerberos authentication, to create the DB cluster in.
+   *
+   * @default - Do not join domain
```

Comment:
This feels a bit vague to me but it could just be that I'm missing some context. Is this saying that the cluster will be created but just not within a domain (or within the Active Directory)?

Reply:
I understand, you're right; the cluster will be created but not associated with any domain. I've revised the comment section for clarity. Could you please check it again?
(I have limited English skills, so I would appreciate it if you could appropriately adjust the text for me.)

Reply:
The new description is a lot clearer to me now! And no worries at all, I really appreciate that you do your best to address all these description changes!